charts/osm-arc/templates/metrics-agent-deployment.yaml

```diff
@@ -16,8 +59,13 @@ spec:
       labels:
         app: osm-metrics-agent
         {{- include "osm.arcLabels" . | nindent 8 }}
+      {{- if (ne .Values.Azure.Cluster.Distribution "openshift") }}
+      annotations:
+        seccomp.security.alpha.kubernetes.io/pod: runtime/default
+      {{- end }}
     spec:
-      serviceAccountName: {{ .Release.Name }}
```

Comment:
this is required by the geneva agent to be set to the extension service account per: https://msazure.visualstudio.com/One/_wiki/wikis/One.wiki/214501/Metric-Agent-Steps?anchor=sample-yaml-files-with-msi-adapter-sidecar-and-mdm-agent

Reply:
@SanyaKochhar @nshankar13 can you research/document the effects of giving the extension service account for openshift privileged escalation?

Reply:
Just tested using that guide ^^ and test metrics are available in Jarvis dashboard.

Reply:
@ksubrmnn yes will do, though I think that would be best to update in a follow up PR for a doc.